src/platform_impl/ios/app_state.rs

```diff
@@ -612,9 +612,9 @@ pub(crate) fn handle_nonuser_events<I: IntoIterator<Item = EventWrapper>>(
         match wrapper {
             EventWrapper::StaticEvent(event) => {
                 if !processing_redraws && event.is_redraw() {
-                    log::info!("processing `RedrawRequested` during the main event loop");
```

Comment:
Made this a `trace` just for debugging. For some reason this is very noisy with bevy.